book/src/package-managers.md

```diff
@@ -23,7 +24,14 @@
 
 The following third party repositories are available:
 
-### Ubuntu
+### Ubuntu/Debian
+
+Install the Debian package from the release page.
+
+If you are running a system older than Ubuntu 22.04, Mint 21, or Debian 12, you can build the `.deb` file locally
+[from source](https://docs.helix-editor.com/building-from-source.html#building-the-debian-package).
```

Comment:
```suggestion
[from source](./building-from-source.md#building-the-debian-package).
```

we can link to the md file and mdbook should take care of making this a relative path, so it should work on `docs.helix-editor.com/` and `docs.helix-editor.com/master` as well as locally (`mdbook serve`)